tests/regression/lp-1844496/task.yaml

```diff
@@ -1,17 +1,34 @@
 summary: regression test for https://bugs.launchpad.net/snapd/+bug/1844496
```

Comment:
will there a new spread test about something similar to the firefox issue? or does this test covers it already but the handling is different?

Reply:
This test does not cover the issue, I will push one here thanks for reminding me

Reply:
Added a spread test here now and also proposed it separately in https://github.com/snapcore/snapd/pull/11530 to show it fails on master